pkg/services/baremetal/client/ssh/ssh_client.go

```diff
@@ -243,12 +243,17 @@ EOF`, cmd))
 		return out
 	}
 
-	out = c.runSSH(`sh /root/install-image-script.sh`)
-	if out.Err != nil {
-		return out
+	installImageOut := c.runSSH(`sh /root/install-image-script.sh`)
+
+	debugTxtOut := c.runSSH(`cat /root/debug.txt`)
+	installImageOut.StdOut = fmt.Sprintf("stdout:\n%s\n###################\n\nstderr:\n%s###################\n\ndebug.txt:\n%s%s",
```

Comment:
should we not have "stdout:" etc. as well as "err:" of the debugTxtOut in the print statement? 
Two problems I see: 
 a) It is not so visible what is stdout and stderr of the cat command. You are the expert and have to say whether that's okay
 b) what if the command does not work? E.g. due to a temporary problem of accessing the server with ssh? Then stdout and maybe stderr will be both empty. In that case we should have the error printed that it is clear why.

Reply:
> a) It is not so visible what is stdout and stderr of the cat command. You are the expert and have to say whether that's okay

I think that's ok, since stderr of `cat /root/debug.txt` should almost always be empty. But I will add a newline between both, so that it is easier to read, if there is stderr.

> b) what if the command does not work? 

Good point. I will add it. But installimage is successful, but getting debug.txt is not? Will this result in returning an err?

Reply:
I think you shouldn't return an error of failing to get debug.txt, but rather print the error  in the above output..

Reply:
> I think you shouldn't return an error of failing to get debug.txt, but rather print the error in the above output..

good point. I updated the PR.